skrifa/src/outline/glyf/mod.rs

```diff
@@ -126,6 +131,7 @@ impl<'a> Outlines<'a> {
             units_per_em: font.head().ok()?.units_per_em(),
             os2_vmetrics,
             prefer_interpreter,
+            prevent_fractional_scaling,
```

Comment:
Google generally prefers positive flags, e.g. enable fractional scaling. This tends to result in more intuitive use, avoids things like "don't prevent X."